vpr/src/route/connection_box_lookahead_map.cpp

```diff
@@ -153,57 +120,39 @@ int CostMap::node_to_segment(int from_node_ind) const {
     return segment_map_[from_node_ind];
 }
 
+static util::Cost_Entry penalize(const util::Cost_Entry& entry, int distance, float penalty) {
+    penalty = std::max(penalty, PENALTY_MIN);
+    return util::Cost_Entry(entry.delay + distance * penalty * PENALTY_FACTOR,
```

Comment:
Consider filling a bug about exploring penalizing base costs too?

Reply:
https://github.com/SymbiFlow/vtr-verilog-to-routing/issues/341